protocol-units/settlement/mcr/client/src/lib.rs

```diff
@@ -4,14 +4,16 @@ use tokio_stream::Stream;
 #[cfg(test)]
 pub mod tests;
 
-#[cfg(feature = "mock")]
 pub mod mock;
 
-// #[cfg(feature = "eth")]
-// pub mod eth;
-#[cfg(feature = "eth")]
+#[cfg(feature = "mock")]
+pub use mock::*;
```

Comment:
The mock can still be imported from top level with nothing in the name indicating that it's a mock: `use mcr_settlement_client::McrSettlementClient`. I think this is confusing and this glob re-export better be removed.

Reply:
Okay, I'll make this change.

Reply:
I think I'll put in the #191  though so that we can update the application.